src/cli/project/export/conda_environment.rs

```diff
@@ -0,0 +1,362 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use itertools::Itertools;
+use miette::{Context, IntoDiagnostic};
+use pep508_rs::ExtraName;
+use pixi_manifest::{
+    pypi::{PyPiPackageName, VersionOrStar},
+    FeaturesExt, HasFeaturesIter, PyPiRequirement,
+};
+use rattler_conda_types::{
+    EnvironmentYaml, MatchSpec, MatchSpecOrSubSection, ParseStrictness, Platform,
+};
+use rattler_lock::FindLinksUrlOrPath;
+
+use crate::project::Environment;
+use crate::Project;
+
+#[derive(Debug, Parser)]
+#[clap(arg_required_else_help = false)]
+pub struct Args {
+    /// Explicit path to export the environment to
+    pub output_path: Option<PathBuf>,
```

Comment:
I like the idea of the default behavior, could you make that for `pixi project export ces`  to? As I would like these interfaces to be similar.

Reply:
Mind if that waits for a separate PR?

Reply:
One question here is the requested design for `ces` was to export by default a file per platform/env. Here there is a single output so without an `output_path` it just prints to stdout. For `ces` what would the behavior be if no output directory is specified given that without a `--platform` or `--env` flag it's going to try to write for all combinations of platform and env?